aml-development/frontend
# feat(subscriptions): add search description

Adds search description.
Adds ability to subscribe to tags and categories from the search
description.

Closes #AMLOS-761

To test:
Search with term and a filter in each filter section.
all should be listed in the descriptions. 
user should be able to see when they are subscribed to tags and categories.
test subscriptions. look at profile to verify subscription
